launcher/modManager/cmodlistview_moc.cpp

```diff
@@ -243,8 +248,11 @@ QString CModListView::genModInfoText(CModEntry & mod)
 	result += replaceIfNotEmpty(mod.getValue("installedVersion"), lineTemplate.arg(tr("Installed version")));
 	result += replaceIfNotEmpty(mod.getValue("latestVersion"), lineTemplate.arg(tr("Latest version")));
 
-	if(mod.getValue("size").isValid())
-		result += replaceIfNotEmpty(CModEntry::sizeToString(mod.getValue("size").toDouble()), lineTemplate.arg(tr("Download size")));
+	if(mod.getValue("localSize").isValid())
+		result += replaceIfNotEmpty(CModEntry::sizeToString(mod.getValue("localSize").toDouble()), lineTemplate.arg(tr("Size")));
+	if((mod.isAvailable() || mod.isUpdateable()) && mod.getValue("size").isValid())
+		result += replaceIfNotEmpty(CModEntry::sizeToString(mbToBytes(mod.getValue("size").toDouble())), lineTemplate.arg(tr("Download size")));
+	
```

Comment:
Perhaps store "localSize" in megabytes as well instead of bytes, for consistency?

Reply:
it's not stored or serialised but internal field. All internal computations are performed in bytes, including qt stuff. So the only conversion now from Mb to bytes is reading size from mod.json
Changing this value will require two additional conventions in few places (b -> mb and back)

Reply:
In that case please name fields explicitly, e.g.
"localSizeBytes"
"downloadSizeMB"